src/radical/pilot/session.py

```diff
@@ -229,7 +229,8 @@ def _init_primary(self):
 
         # if user did not set a uid, we need to generate a new ID
         if not self._uid:
-            self._uid = ru.generate_id('rp.session', mode=ru.ID_PRIVATE)
+            base = os.environ.get('RP_SID_BASE', 'rp.session')
```

Comment:
quick check on naming - env variables which start with `RP_` are internal ones, but the ones that start with `RADICAL_PILOT_` are set by users (aka external) - is this one an internal variable?

p.s., one more thing - `BASE` -> `PREFIX`?

Reply:
Ack on the var name, will change.  I think `BASE` is ok as it is not a prefix to the 'normal' session ID, but a completely different base name.

Reply:
Thanks - I also fixes the actual `generate_uid` call - that embarrassingly did not yet yield what the doc string promised...